bootstrap.py

```diff
@@ -639,6 +639,26 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = "katello-capsule"
+    check_rpm_installed(rpm_sat)
+
+	
+def check_rpm_installed(rpm_sat):
+    ts = rpm.TransactionSet()
+    headers = ts.dbMatch('name', rpm_sat)
```

Comment:
It may make sense to check against a list of packages, not just a single package, as depending on configuration (and upstream vs. downstream) the list of packages that define Katello or Satellite may be > 1. This could also keep from repeating `headers = ts.dbMatch('name', rpm_sat)` for each package. 
 
Example:
~~~
    rpm_sat = ['katello-capsule','satellite']
    check_rpm_installed(rpm_sat)

def check_rpm_installed(rpm_sat):
    ts = rpm.TransactionSet()
    headers = ts.dbMatch()

    for h in headers:
        if h['name'] in rpm_sat:
            print_error("RPM Satellite/capsule installed")
            sys.exit(1)

~~~